test/Dialect/FIRRTL/canonicalization.mlir

```diff
@@ -3514,6 +3514,73 @@ firrtl.module @HasBeenReset(in %clock: !firrtl.clock, in %reset1: !firrtl.uint<1
   %constClockR1 = firrtl.node sym @constClockR1 %c5 : !firrtl.uint<1>
 }
 
+// OMIR annotations should not block removal.
+//   - See: https://github.com/llvm/circt/issues/6199
+//
+// CHECK-LABEL: firrtl.module @OMIRRemoval
+firrtl.module @OMIRRemoval(in %source : !firrtl.uint<1>) {
+  // CHECK-NOT: %tmp_0
+  %tmp_0 = firrtl.wire {
+    annotations = [
+      {
+         class = "freechips.rocketchip.objectmodel.OMIRTracker",
```

Comment:
All of the tests will need to be updated from `"freechips.rocketchip.objectmodel.OMIRTracker"` to `"circt.tracker"`.